services/blockchain-connector/config.js

```diff
@@ -41,24 +42,29 @@ config.liskAppDataPath = process.env.LISK_APP_DATA_PATH || '~/.lisk/lisk-core';
 /**
   * Network-related settings
   */
-config.genesisBlockUrl = process.env.GENESIS_BLOCK_URL || '';
-config.networks = [
-	{
-		name: 'mainnet',
-		identifier: '4c09e6a781fc4c7bdb936ee815de8f94190f8a7519becd9de2081832be309a99',
-		genesisBlockUrl: 'https://downloads.lisk.com/lisk/mainnet/genesis_block.json.tar.gz',
-	},
-	{
-		name: 'testnet',
-		identifier: '15f0dacc1060e91818224a94286b13aa04279c640bd5d6f193182031d133df7c',
-		genesisBlockUrl: 'https://downloads.lisk.com/lisk/testnet/genesis_block.json.tar.gz',
-	},
-	{
-		name: 'betanet',
-		identifier: '15f0dacc1060e91818224a94286b13aa04279c640bd5d6f193182031d133df7c',
-		genesisBlockUrl: 'https://downloads.lisk.com/lisk/betanet/genesis_block.json.tar.gz',
-	},
-];
+config.constants.GENESIS_BLOCK_URL_DEFAULT = '';
+config.genesisBlockUrl = process.env.GENESIS_BLOCK_URL
+							|| config.constants.GENESIS_BLOCK_URL_DEFAULT;
```

Comment:
Please fix the alignment.
```suggestion
config.constants.GENESIS_BLOCK_URL_DEFAULT = '';
config.genesisBlockUrl = process.env.GENESIS_BLOCK_URL || config.constants.GENESIS_BLOCK_URL_DEFAULT;
```

Reply:
resolved in offline discussion.